examples/create-react-app-with-styled-components-engine/src/App.js

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import Slider from '@material-ui/lab/SliderStyled';
+import Typography from '@material-ui/core/Typography';
+
+export default function App() {
+  return (
+    <div>
+      <Typography>Styled slider powered by styled-components</Typography>
+      <Slider defaultValue={30} aria-labelledby="primary-slider" color="primary" />
```

Comment:
Do these `aria-labelledby` attributes reference anything?

Reply:
Oops I think I just copied and stripped the example. Will remove them